modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java

```diff
@@ -21,8 +21,8 @@
  */
 public class ToxiproxyContainer extends GenericContainer<ToxiproxyContainer> {
 
-    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("shopify/toxiproxy");
-    private static final String DEFAULT_TAG = "2.1.0";
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("ghcr.io/shopify/toxiproxy");
```

Comment:
we need to keep backward compatibility, in this case I would suggest to add a new image name

```java
private static final DockerImageName GHCR_IMAGE_NAME = DockerImageName.parse("ghcr.io/shopify/toxiproxy");
```

and add it [here](https://github.com/testcontainers/testcontainers-java/blob/de1324ed2800eff4da326d0c23d281399d006bc0/modules/toxiproxy/src/main/java/org/testcontainers/containers/ToxiproxyContainer.java#L56)

```java
dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME, GHCR_IMAGE_NAME);
```